### PR TITLE
reorganizing the ToC

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: docs/content/toc.md [hunk 1]
```diff
@@ -126,20 +132,19 @@ layout: toc
...
-      * [Historical](/docs/VERSION/operations/api-reference.html#historical)  
+      * [Historical](/docs/VERSION/operations/api-reference.html#historical)
+  * Tuning and Recommendations
+    * [Basic Cluster Tuning](/docs/VERSION/operations/basic-cluster-tuning.html)  
+    * [General Recommendations](/docs/VERSION/operations/recommendations.html)
+    * [Performance FAQ](/docs/VERSION/operations/performance-faq.html)
+    * [JVM Best Practices](/docs/VERSION/configuration/index.html#jvm-configuration-best-practices)        
   * Tools
     * [Dump Segment Tool](/docs/VERSION/operations/dump-segment.html)
     * [Insert Segment Tool](/docs/VERSION/operations/insert-segment-to-db.html)
```
Comment (jihoonson) [inline docs/content/toc.md]: Would you please remove this line? It was removed in https://github.com/apache/incubator-druid/pull/6911.

Comment (fjy) [reply docs/content/toc.md]: I don't see insert segment tool removed. Moreover, @gianm added it back in in #7658

Comment (fjy) [reply docs/content/toc.md]: I'm fine removing it as long as it is the right thing to do

Comment (jihoonson) [reply docs/content/toc.md]: Ah I see. It looks fine to me.

**Comment by fjy** [comment]
@jihoonson do you mind helping me backport this to 0.15?

**Comment by jihoonson** [comment]
Sure, raised https://github.com/apache/incubator-druid/pull/7739.

**Pull request merged**